doc/lua_api.txt

```diff
@@ -3916,6 +3916,13 @@ The Biome API is still in an experimental phase and subject to change.
     -- ^ Minimum and maximum `y` positions these decorations can be generated at.
     -- ^ This parameter refers to the `y` position of the decoration base, so
     --   the actual maximum height would be `height_max + size.Y`.
+        spawn_by = "default:water",
+    --  ^ Node that the decoration only spawns next to.
+    --  ^ The neighbours checked are the 16 nodes level with and surrounding
+    --  ^ these 2 nodes: the surface node and the node above the surface node.
```

Comment:
Oh you're right, okay that makes total sense then.

So use my wording but maybe add "two horizontal planes" and obviously correct that to surface + 1.

Yeah, makes total sense now.